src/input.cpp

```diff
@@ -510,6 +510,10 @@ void input_manager::init_keycode_mapping()
     add_keyboard_char_keycode_pair( KEY_END,       translate_marker_context( "key name", "END" ) );
     add_keyboard_char_keycode_pair( '\n',          translate_marker_context( "key name", "RETURN" ) );
 
+    for( char c = 'A'; c <= 'Z'; c++ ) {
+        add_keyboard_char_keycode_pair( c - 64, string_format( "CTRL+%s", c ) );
```

Comment:
```suggestion
        add_keyboard_char_keycode_pair( c - 64, string_format( _( "CTRL+%s" ), c ) );
```

Reply:
Translation of this string should be handled in `get_keyname`, see my comments below.